backbone.js

```diff
@@ -1440,6 +1440,11 @@
       return path === this.root && !this.getSearch();
     },
 
+    // Decode unicode escapes without decoding `%25`.
```

Comment:
Can you add a bit more prose here for future-usses who might look at this quizzically? It's always nice to have a bit more context in comments than just looking at the code would give us.